lib/puppet/pops/model/model_label_provider.rb

```diff
@@ -60,6 +60,7 @@ def label_ConcatenatedString o          ; "Double Quoted String"              en
   def label_HeredocExpression o           ; "'@(#{o.syntax})' expression"       end
   def label_HostClassDefinition o         ; "Host Class Definition"             end
   def label_FunctionDefinition o          ; "Function Definition"               end
+  def label_PLanDefinition o              ; "Plan Definition"                   end
```

Comment:
this typo means that the method isn't found (L is uppercase)

Reply:
meh